miso-web/src/main/webapp/scripts/hot.js

```diff
@@ -791,7 +798,44 @@ var HotUtils = {
         obj[property] = flat[property];
       }
     }
-  }
+  },
+  
+  makeQcActions : function(qcTarget) {
+    return [
+        null,
+        {
+          name : 'Add QCs',
+          action : function(items) {
+            Utils
+                .showDialog(
+                    'Add QCs',
+                    'Add',
+                    [ {
+                      property : 'copies',
+                      type : 'int',
+                      label : 'Copies per ' + qcTarget,
```

Comment:
Sorry, but I really think that `copies` suffers from the same issue as `replicates` -- it implies a near identicalness (photocopy, copy-paste), whereas the lab will be adding "one Qubit, one qPCR, one Tape Station".